NetworkingTests/Tests/LoginRepositorySpec.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  LoginRepositorySpec.swift
+//  Networking
+//
+//  Created by Pablo Giorgi on 5/5/17.
+//  Copyright © 2017 Wolox. All rights reserved.
+//
+
+import Quick
+import Nimble
+import Networking
+
+internal class LoginRepositorySpec: QuickSpec {
+    
+    override func spec() {
+        
+        var repository: LoginRepositoryType!
+        
+        beforeEach() {
+            let networkingConfiguration = NetworkingConfiguration(useSecureConnection: true,
+                                                                  domainURL: "localhost",
+                                                                  port: 8080,
+                                                                  subdomainURL: "/local-path-1.0",
+                                                                  usePinningCertificate: false)
+            
+            repository = LoginRepository(networkingConfiguration:networkingConfiguration,
+                                         requestExecutor: LocalRequestExecutor(),
+                                         sessionManager: SessionManagerMock())
+        }
+        
+        describe("#login") {
+            
```

Comment:
It's the same than when any other request fails, anyway I added a test for this.